transform_aes.c

```diff
@@ -76,9 +90,10 @@ static void set_aes_cbc_iv(transop_aes_t *priv, n2n_aes_ivec_t ivec, uint64_t iv
  *
  *  - a 8-bit aes encoding version in clear text
  *  - a 64-bit random IV seed
- *  - encrypted payload.
+ *  - a 8-bit compression indicating field; encrypted, together with the following...
```

Comment:
It's better to have a generic "encrypted flags" fields, which will give us future extensibility

Reply:
Okay. I guess this should be regular part of packet spec then. I will rename the "compression indicating field" in the code to something more genereic. Is 8-bit size okay? Or shall we go for 16?

Reply:
8 bits are fine for now

Reply:
I renamed the field and the makros accordingly. Also, I adopted the comments.